coalib/results/Diff.py

```diff
@@ -200,9 +225,12 @@ def unified_diff(self):
         for equality comparison.
         """
         return ''.join(difflib.unified_diff(
-            self.original,
-            self.modified,
-            tofile=self.rename if isinstance(self.rename, str) else ''))
+                       self.original,
+                       self.modified,
+                       tofile=self.rename if isinstance(self.rename, str)
+                       else self.create
+                       if isinstance(self.create, str)
+                       else ''))
```

Comment:
Do you like the new one? I had to put the brackets to be allowed to use one more level of indent